src/Compilers/CSharp/Test/Symbol/Symbols/LocalFunctionTests.cs

```diff
@@ -189,5 +189,181 @@ public void M()
                 // }
                 Diagnostic(ErrorCode.ERR_EOFExpected, "}").WithLocation(7, 1));
         }
+
+        [Fact, WorkItem(65938, "https://github.com/dotnet/roslyn/issues/65938")]
+        public void StaticLocalFunction_CapturingMethodGroup()
+        {
+            CreateCompilation("""
+                using System;
+
+                var c = new C();
+                LocalFunc();
+                NonStatic();
+
+                static void LocalFunc()
+                {
+                    var x = c.MyExtension;
```

Comment:
>x [](http://example.com/codeflow?start=24&length=1)

Consider naming the locals distinctly across the static local function and static lambda expression below, so it's easier to match up diagnostics.